zoopla/api_v1.py

```diff
@@ -92,6 +95,11 @@ def finished():
                 L.debug("Stop paging, yielded={}, result_count={}".format(
                     num_yielded, result_count))
                 return True
+            elif args['page_number'] == 100:
```

Comment:
This should be >= 100 if for example someone picks a specific page number > 100 rather than simply incrementing.